src/main/java/nextstep/subway/application/LineService.java

```diff
@@ -0,0 +1,68 @@
+package nextstep.subway.application;
+
+import java.util.List;
+import java.util.stream.Collectors;
+import javax.persistence.NoResultException;
+import nextstep.subway.domain.Line;
+import nextstep.subway.domain.Station;
+import nextstep.subway.dto.LineRequest;
+import nextstep.subway.dto.LineResponse;
+import nextstep.subway.repository.LineRepository;
+import nextstep.subway.repository.StationRepository;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+@Service
+@Transactional(readOnly = true)
+public class LineService {
+
+    private LineRepository lineRepository;
```

Comment:
큰 상관은 없지만, 컨벤션상 보통 final 필드로 선언해서 생성자 주입을 선호하는 것 같아요 :)

```suggestion
    private final LineRepository lineRepository;
```

Reply:
넵 다음 미션 때 반영하겠습니다!